### PR TITLE
Remove redis & add error-cat 

### PR DESCRIPTION
**Pull request by podviaznikov**
1. remove redis
2. add error-cat
3. log rabbit error
4. update loadenv & monitordog
5. add blacklisted container to ignore weave utility container

### CODE REVIEW

File: test/listener.js [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
 var noop = require('101/noop');
+var errorCat = require('error-cat');
 
+var sinon = require('sinon');
 var docker = require('./fixtures/docker-mock');
 var Listener = require('../lib/listener');
 var status = require('../lib/status');
 
+
+
```
Comment (bkendall) [inline test/listener.js]: /me dislikes whitespace...

**Comment by cflynn07** [comment]
LGTM

File: configs/.env [hunk 2]
```diff
@@ -1,7 +1,7 @@
 AUTO_RECONNECT=true
+CONTAINERS_BLACKLIST=weaveworks/weaveexec
```
Comment (anandkumarpatel) [inline configs/.env]: lets add `swarm` too

File: lib/publisher.js [hunk 3]
```diff
@@ -27,18 +25,31 @@ function isBuildContainer (enhanced) {
 /**
  * @class
  */
-function RedisPublisherStream () {
+function RabbitPublisherStream () {
   stream.Writable.call(this);
 }
 
-util.inherits(RedisPublisherStream, stream.Writable);
+// Ignore events from blacklisted conatiners
```
Comment (anandkumarpatel) [inline lib/publisher.js]: s/conatiners/containers/g

**Comment by anandkumarpatel** [comment]
looks good to me!

File: configs/.env [hunk 4]
```diff
@@ -1,7 +1,7 @@
 AUTO_RECONNECT=true
+CONTAINERS_BLACKLIST=weaveworks/weaveexec, swarm
```
Comment (anandkumarpatel) [inline configs/.env]: space?

**Pull request merged**